paket.dependencies

```diff
@@ -1,24 +1,32 @@
 redirects: on
+storage: none
 source https://www.nuget.org/api/v2/
 
-// We want to depend on FSharp.Core 4.0.0.1, but not require
-// that version in the generated nuget package (see paket.template).
+// We want to depend on FSharp.Core 4.1.17 (it's the earliest one
+// that supports .NET standard), but not require that version in
+// the generated nuget package (see paket.template).
 //
-// If we just specify "FSharp.Core 4.0.0.1" then the nuget package
+// If we just specify "FSharp.Core 4.1.17" then the nuget package
 // has an dependency on that *exact* version.
 //
-// So, we specify 4.0.0.1 as a minimum bound, and set lowest_matching
+// So, we specify 4.1.17 as a minimum bound, and set lowest_matching
 // so it won't be updated automatically to a newer one by 'paket update'.
-nuget FSharp.Core >= 4.0.0.1 lowest_matching: true
+nuget FSharp.Core >= 4.1.17 lowest_matching: true
 
-nuget FAKE 4.41.1
-nuget xunit.runner.console 2.1.0
-nuget xunit.runner.visualstudio 2.1.0 version_in_path: true
-nuget xunit.assert 2.1.0
-nuget xunit.core 2.1.0
+// Similar reasoning applies here (this is used by the FSharpx collections
+// below):
+
+nuget System.Collections.Specialized >= 4.0.1 lowest_matching: true
+
+// These files are imported directly into Hedgehog:
 
 github fsprojects/FSharpx.Collections:e2722ce571f4926586ae907413b4a5511f0ef74e src/FSharpx.Collections/Collections.fs
 github fsprojects/FSharpx.Collections:e2722ce571f4926586ae907413b4a5511f0ef74e src/FSharpx.Collections/LazyList.fsi
 github fsprojects/FSharpx.Collections:e2722ce571f4926586ae907413b4a5511f0ef74e src/FSharpx.Collections/LazyList.fs
 
-nuget Unquote 3.1.2
+// For test projects:
```

Comment:
Instead of comments, I think we can use Paket's (dependency groups)[https://fsprojects.github.io/Paket/groups.html] feature. This can be done later though (so we don't hold the PR just for this).

Reply:
I did think about this but it seemed like more hassle than needed... e.g. in the `paket.references` for the projects you have to use the same groups.

Reply:
Sounds reasonable. Let's leave it as-is for now.